playbooks/ci_restore_snapshot.yaml

```diff
@@ -7,15 +7,24 @@
   hosts: cluster_machines
   become: true
   tasks:
+    - name: "grub conf"
```

Comment:
this task is already present in the debian roles, is it really useful to do it here ?

Reply:
yes, the idea here is that we are rollbacking the machines to zero. The LVM rollback only applies to vg1/root, and the grub configuration is in /boot, so we need this task to rollback the grub config.